static/test/javascripts/spec/common/experiments/ab.spec.js

```diff
@@ -297,6 +298,9 @@ define([
             it('should fire the success function when canRun is false', function() {
                 var spy = sinon.spy();
 
+                ab.addTest(test.one);
+                ab.segmentUser();
```

Comment:
This is just required to fix the test.  In the real world, a user will always be segmented before the success event runs.  The test previously didn't segment the user, but because of the fix (https://github.com/guardian/frontend/pull/15714#discussion-diff-98672710R250), it needs to.